### PR TITLE
NuTemplate

### PR DESCRIPTION
**Pull request by croguy**
This kills the bad structuring.

### CODE REVIEW

File: FP_NuTemplate_Cro.VR/config.sqf [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Only allow people with map tools to draw on briefing (true = everyone can draw)
+ACE_maptools_EveryoneCanDrawOnBriefing = false;
```
Comment (ComfyPillow) [inline FP_NuTemplate_Cro.VR/config.sqf:2]: Didn't this variable and the ACE Linedrawing get removed in favor of the Vanilla functionality?

**Comment by Cuel** [comment]
Why is it its own folder?

**Comment by croguy** [comment]
So we can keep the regular one if you want to alter it in a more radical manner.

But if you want to we could easily just triple check if it works as intended and replace the standard one.

**Comment by Cuel** [comment]
There's no point having two folders. It's hard to see what's changed, want me to close this  request so you can make a new one with cleaner history?

**Comment by Cuel** [comment]
@croguy ?

**Comment by croguy** [comment]
@Cuel Gotcha. I was extremely busy so I put off from working on the framework itself.

Go ahead with the request closing. Do you want a new variant where the old framework has been completely changed?

**Comment by Cuel** [comment]
Yeah only with the changes you've made. It's to track changes.

**Pull request closed without merging**